shared/params.go

```diff
@@ -18,6 +18,44 @@ import (
 	mapset "github.com/deckarep/golang-set"
 )
 
+// TestRunFilter represents the ways TestRun entities can be filtered in
+// the webapp and api.
+type TestRunFilter struct {
+	SHA      string
+	Labels   mapset.Set
+	Complete bool
+	MaxCount *int
+	Products []Product
+}
+
+// ToQuery converts the filter set to a url.Values (set of query params).
+// defaultQuery is whether the params should fall back to the default query
+// (on the homepage).
+func (filter TestRunFilter) ToQuery(defaultQuery bool) (q url.Values) {
```

Comment:
IMHO instead of `defaultQuery` we should call it `complete`. And in the future if we have more "default" filtering on the homepage we should add separate boolean parameters for each one of them. WDYT?

Reply:
Sure.